scripts/GetClrDbg.ps1

```diff
@@ -173,26 +122,16 @@ if (IsLatest $InstallPath $RuntimeID $VersionNumber) {
         }
     }
     
-    Push-Location $TempPath -ErrorAction Stop
+    $target = ("clrdbg-" + $VersionNumber).Replace('.','-') + "/clrdbg-" + $RuntimeID + ".zip"
+    $url = "https://vsdebugger.azureedge.net/" + $target
     
-    Write-Host "Info: Generating project.json"
-    GenerateProjectJson $VersionNumber $RuntimeID
-
-    Write-Host "Info: Generating NuGet.config"
-    GenerateNuGetConfig
-
-    Write-Host "Info: Executing dotnet restore"
-    dotnet restore
-
-    Write-Host "Info: Executing dotnet publish"
-    dotnet publish -r $RuntimeID -o $InstallPath
-
-    Pop-Location
-
-    Remove-Item -Path $TempPath -Recurse -Force
+    if (Test-Path $InstallPath) {
+        Remove-Item -Path $InstallPath -Force -Recurse
+    }
+    
+    DownloadAndExtract $url $InstallPath
 
     WriteSuccessInfo $InstallPath $RuntimeID $VersionNumber
     Write-Host "Info: Successfully installed clrdbg at '$InstallPath'"
 }
 
```

Comment:
Do you know if the Docker folks fixed their issue with extra bytes at the end of the script?